docs/source/dataformat.rst

```diff
@@ -744,24 +793,181 @@ According to the table above, a song containing two verses (*v1, v2*), a chorus
 Chords
 ^^^^^^
 
-The OpenLyrics format also provides the ability to include chords in the lyrics of
-songs. The tag containing a chord name looks like this::
-
-    <chord name="D7"/>
-
-The ``<chord>`` tags are mixed in with the lyrics of a song. This tag should be
-placed immediately before the letters where it should be played::
+The OpenLyrics format also provides the ability to include chords in the lyrics and instrumental part of
+songs. The tag containing a chord name looks like these::
+
+    <chord root="C" structure="dom7">lyrics...</chord>
+    <chord root="D" bass="F#">lyrics...</chord>
+    <chord root="C" structure="min" bass="Eb"/>lyrics
+    <chord root="E" structure="3-5-m7-13">lyrics...</chord>
+
+The root note
+"""""""""""""
+
+The ``root`` attribute describes the root note of the chord. The values should marked
+with English notation:
+
+========== === ===== ===== === ===== ==== === === ===== ===== ==== ===== ==== === ===== ==== ===
+english    C   C#    Db    D   D#    Eb   E   F   F#    Gb    G    G#    Ab   A   A#    Bb   B
+========== === ===== ===== === ===== ==== === === ===== ===== ==== ===== ==== === ===== ==== ===
+german     C   Cis   Des   D   Dis   Es   E   F   Fis   Ges   G    Gis   As   A   Ais   B    H
+dutch      C   Cis   Des   D   Dis   Es   E   F   Fis   Ges   G    Gis   As   A   Ais   Bes  B
+hungarian  C   Cisz  Desz  D   Disz  Esz  E   F   Fisz  Gesz  G    Gisz  Asz  A   Aisz  B    H
+neolatin   Do  Do#   Reb   Re  Re#   Mib  Mi  Fa  Fa#   Solb  Sol  Sol#  Lab  La  La#   Sib  Si
+========== === ===== ===== === ===== ==== === === ===== ===== ==== ===== ==== === ===== ==== ===
+
+The preferred notation for displaying can be marked with ``chordNotation`` attribute on root element.
+
+The bass
+""""""""
+
+The optional ``bass`` attribute describes the foreign bass of the chord if any. The values should marked
+with English notation.
+
+The chord structure
+"""""""""""""""""""
+
+The ``structure`` attribute describes the kind of the chord. This element is optional,
+if not present, de default value is the ``major``. It can be marked
+
```

Comment:
Remove this line

Reply:
Raoul, thanks for corrections. I added them all. I am very grateful because English is not one of my strength.

But this is unclear for me. What is you suggestion?

a) 
```
The chord structure
"""""""""""""""""""
The ``structure`` attribute describes the kind of the chord. This element is optional,
```
b)
```
if not present, de default value is the ``major``. It can be marked
- with a sorthand code, or
```

Reply:
@gyuris My guess is that @rsnyman meant remove the empty line.

Reply:
OK. Done.